pkgs/by-name/gi/git-repo-manager/package.nix

```
@@ -0,0 +1,36 @@
+{
+  pkgs,
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+  openssl,
+  pkg-config,
+}:
+
+rustPlatform.buildRustPackage rec {
+
+  pname = "git-repo-manager";
+  version = "0.7.16";
+
+  src = fetchFromGitHub {
+    owner = "hakoerber";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-v8fgTrvgaYQzVDoxfREdkCR/1Kjv4f4mKlCLxXOjkHk=";
+  };
+
+  cargoSha256 = "sha256-APq8qq9B+GOyFiA0uquSB+e8SL1sgxVn6+ZOW2gnZHE=";
```

```suggestion
  useFetchCargoVendor = true;
  cargoHash = "";
```
Evaluation failed since cargoSha256 is deprecated